splitgraph/cloud/__init__.py

```diff
@@ -633,7 +696,7 @@ def _gql(self, query: Dict, endpoint=None, handle_errors=False) -> requests.Resp
         return result
 
     @staticmethod
-    def _prepare_upsert_metadata_gql(namespace: str, repository: str, metadata: Metadata, v1=False):
+    def _validate_metadata(namespace: str, repository: str, metadata: Metadata):
```

Comment:
NB: mypy will ignore this function since it doesn't have a return type (for None-returning functions it can just be `-> None`). `check_untyped_defs = True` in mypy will make it descend into these. I mean to do a pass later on after enabling it (it causes a few dozen type errors), so nothing to do here.